src/modules/python/pbs/v1/_pmi_sgi.py

```diff
@@ -52,8 +52,8 @@
 if pbsexec is None:
     raise BackendError("PBS_EXEC not found")
 
-sys.path.append(os.path.join(pbsexec, "python", "lib", "python2.7"))
-sys.path.append(os.path.join(pbsexec, "python", "lib", "python2.7",
+sys.path.append(os.path.join(pbsexec, "python", "lib", "python3.6"))
```

Comment:
this change will only work for python3.6 but we say 'python >= 3.5' in pbspro.spec file.
I mean we can't hardcode 3.6 in python version number, we have to programmatically find out version and use it if we want to support >= 3.5